src/webportal/src/app/home/home/abnormal-job-list.jsx

```diff
@@ -70,9 +71,9 @@ const AbnormalJobList = ({ jobs, style }) => {
       onRender(job) {
         const { legacy, name, namespace, username } = job;
         const href = legacy
-          ? `/job-detail.html?jobName=${name}`
+          ? `/job-detail.html?jobname=${name}`
```

Comment:
The camelCase `jobName` and lowercase `username` is defined in rest api. I think we need to keep the consistency here